### ISSUE TITLE
Disable AeroSpace on a per-native Space basis 

### ISSUE THREAD

**Issue by vassilmladenov** [nikitabobko/AeroSpace#256]
I read through the docs about the interaction with native Spaces.

> The intended workflow of using AeroSpace workspaces is to only have one macOS Space (or as many monitors you have, if Displays have separate Spaces is enabled) and don’t interact with macOS Spaces anymore.

Could it be possible to disable AeroSpace (e.g. ⌘E) for a specific native Space, so I can have some spaces tiled and some with floating windows? This would help me personally as I've bounced off of most tiling WMs I've tried due to the aggressive window management, and I can stick apps that are less flexible about window sizing in their own floating space.

edit: one example is Messenger and Discord apps, neither of which can get narrow enough to correctly tile on the 1440x932 resolution of a 15" Air, I'd rather just shove those in a floating space off somewhere.

**Comment by nikitabobko** [comment]
> Could it be possible to disable AeroSpace (e.g. ⌘E) for a specific native Space

It's not possible to know what specific Space is currently active without using private APIs (which goes against values of the project)

> and I can stick apps that are less flexible about window sizing in their own floating space.

Instead of dedicated Space for floating windows you can create yourself a dedicated workspace for floating windows.

```toml
alt-shift-2 = ['layout floating', 'move-node-to-workspace 2']

# Tile windows back
alt-shift-1 = ['layout tiling', 'mode-node-to-workspace 1']
alt-shift-3 = ['layout tiling', 'mode-node-to-workspace 3']
alt-shift-4 = ['layout tiling', 'mode-node-to-workspace 4']
# etc
```

Now you can use `alt-shift-2` shortcut to float the window and send it to workspace 2, which effectively makes your second workspace a workspace for floating windows

---

This won't work as expected when you open new windows while workspace 2 is already active (you'd need to manually press `alt-shift-2` every time you open a new window), I think it could be fixed by introducing one more condition in `on-window-detected` callback:

```toml
[[on-window-detected]]
if.workspace = '2'
run = 'layout floating'
```

Since I think it's a good idea, which is easy to implement, I introduced `if.workpsace` in `main` branch https://github.com/nikitabobko/AeroSpace/commit/d68f1deae34e19e36cc9956730a2e54a47b3bc9f

It will become available in the next update, though you can build from sources, if you eager to try it

**Comment by nikitabobko** [comment]
If AeroSpace will ever introduce dynamic twm capabilities #260, the desired workflow will be even easier to support by using `floating` normalization

**Comment by nikitabobko** [comment]
> Since I think it's a good idea, which is easy to implement, I introduced if.workpsace in main branch https://github.com/nikitabobko/AeroSpace/commit/d68f1deae34e19e36cc9956730a2e54a47b3bc9f
> 
> It will become available in the next update, though you can build from sources, if you eager to try it

Released in 0.12.0-Beta
